github/orgs_audit_log.go

```diff
@@ -101,16 +101,16 @@ func (a *AuditEntry) GetOrgID() (orgID int64, ok bool) {
 	return orgID, true
 }
 
-// GetOrgIDSlice returns the OrgID field, as a slice of int64, if it's valid.
-func (a *AuditEntry) GetOrgIDSlice() (orgID []int64, ok bool) {
+// GetOrgIDInt64s returns the OrgID field, as a slice of int64, if it's valid.
```

Comment:
Now that we don't have `GetOrgIDStrings`, this might be much nicer as `GetOrgIDs`.